actiontext/test/unit/model_encryption_test.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class ActionText::ModelEncryptionTest < ActiveSupport::TestCase
+  test "encrypt content based on :encrypted option at declaration time" do
+    encrypted_message = EncryptedMessage.create!(subject: "Greetings", content: "Hey there")
+    assert_encrypted_rich_text_attribute encrypted_message, :content, "Hey there"
+
+    clear_message = Message.create!(subject: "Greetings", content: "Hey there")
+    assert_not_encrypted_rich_text_attribute clear_message, :content, "Hey there"
+  end
+
+  test "include rich text attributes when encrypting the model" do
+    content = "<p>the space force is here, we are safe now!</p>"
+
+    message = ActiveRecord::Encryption.without_encryption do
+      EncryptedMessage.create!(subject: "Greetings", content: content)
+    end
+
+    message.encrypt
+
+    assert_encrypted_rich_text_attribute(message, :content, content)
+  end
+
+  test "encrypts lets you skip rich texts when encrypting" do
+    content = "<p>the space force is here, we are safe now!</p>"
+
+    message = ActiveRecord::Encryption.without_encryption do
+      EncryptedMessage.create!(subject: "Greetings", content: content)
+    end
+
+    message.encrypt(skip_rich_texts: true)
+
+    assert_not_encrypted_rich_text_attribute(message, :content, content)
+  end
+
+  private
+    def assert_encrypted_rich_text_attribute(model, attribute_name, expected_value)
```

Comment:
Could some form of these two helpers be extracted into a consumer-facing module, so that similar assertions could be made from application test suites?

Reply:
Yes @seanpdoyle I think that makes a lot of sense sense. Most people shouldn't need those, but if you are building a system to mass-encrypt (or re-encrypt) data, those would be very useful 👍. Not sure is I'll get to it in this PR but, if not, it's a great candidate for a future iteration.